looks like a `product.AddFulfillmentPlacesTest` failed:

```
[ERROR] Failures:
[ERROR]   AddFulfillmentPlacesTest.testAddFulfillment:68 expected to contain:
    Add fulfillment places, wait 30 seconds
but was:
    Create product request: parent: "projects/java-docs-samples-testing/locations/global/catalogs/default_catalog/branches/0"
    product {
      type: PRIMARY
      categories: "Speakers and displays"
      title: "Nest Mini"
      brands: "Google"
      price_info {
        currency_code: "USD"
        price: 30.0
        original_price: 35.5
      }
      availability: IN_STOCK
      fulfillment_info {
        type: "pickup-in-store"
        place_ids: "store0"
        place_ids: "store1"
      }
    }
    product_id: "76b33266-346c-4e35-aaef-d17600976703"

    Created product: name: "projects/779844219229/locations/global/catalogs/default_catalog/branches/0/products/76b33266-346c-4e35-aaef-d17600976703"
    id: "76b33266-346c-4e35-aaef-d17600976703"
    type: PRIMARY
    primary_product_id: "76b33266-346c-4e35-aaef-d17600976703"
    categories: "Speakers and displays"
    title: "Nest Mini"
    brands: "Google"
    price_info {
      currency_code: "USD"
      price: 30.0
      original_price: 35.5
    }
    availability: IN_STOCK
    fulfillment_info {
      type: "pickup-in-store"
      place_ids: "store0"
      place_ids: "store1"
    }

    Add fulfilment places with current date: seconds: 1659983885
    nanos: 412000000
    Add fulfillment request product: "projects/java-docs-samples-testing/locations/global/catalogs/default_catalog/branches/0/products/76b33266-346c-4e35-aaef-d17600976703"
    type: "pickup-in-store"
    place_ids: "store2"
    add_time {
      seconds: 1659983885
      nanos: 412000000
    }
    allow_missing: true

    Add fulfillment places, wait 45 seconds:
    Get product response: name: "projects/779844219229/locations/global/catalogs/default_catalog/branches/0/products/76b33266-346c-4e35-aaef-d17600976703"
    id: "76b33266-346c-4e35-aaef-d17600976703"
    type: PRIMARY
    primary_product_id: "76b33266-346c-4e35-aaef-d17600976703"
    categories: "Speakers and displays"
    title: "Nest Mini"
    brands: "Google"
    price_info {
      currency_code: "USD"
      price: 30.0
      original_price: 35.5
      price_range {
      }
    }
    availability: IN_STOCK
    fulfillment_info {
      type: "pickup-in-store"
      place_ids: "store1"
      place_ids: "store0"
    }

    Delete product request name: "projects/java-docs-samples-testing/locations/global/catalogs/default_catalog/branches/0/products/76b33266-346c-4e35-aaef-d17600976703"

    Product projects/java-docs-samples-testing/locations/global/catalogs/default_catalog/branches/0/products/76b33266-346c-4e35-aaef-d17600976703 was deleted.
```

--- a/samples/interactive-tutorials/src/test/java/product/AddFulfillmentPlacesTest.java
+++ b/samples/interactive-tutorials/src/test/java/product/AddFulfillmentPlacesTest.java
@@ -65,7 +65,7 @@ public class AddFulfillmentPlacesTest {
     String outputResult = bout.toString();
 
     assertThat(outputResult).contains("Add fulfilment places with current date");
-    assertThat(outputResult).contains("Add fulfillment places, wait 30 seconds");
+    assertThat(outputResult).contains("Add fulfillment places, wait 45 seconds");
   }
 
   @After

